packages/keyring-controller/tests/mocks/mockEncryptor.ts

```diff
@@ -88,8 +108,18 @@ export default class MockEncryptor implements ExportableKeyEncryptor {
   isVaultUpdated(_vault: string) {
     return true;
   }
+}
 
-  generateSalt() {
-    return MOCK_ENCRYPTION_SALT;
-  }
+function generateSalt() {
+  // Generate random salt.
+
+  // return crypto.randomUUID();
+  return SALT; // TODO some tests rely on fixed salt, but wouldn't it be better to generate random value here?
```

Comment:
Indeed, a more realistic mock encryptor would certainly have advantages. I wonder if we could take it a step further even than making the mock more 'realistic', and instead use a _real_ encryptor that uses some very fast/insecure encryption strategies.